plugins/eos-blacklist/gs-plugin-eos-blacklist.c

```diff
@@ -198,6 +216,7 @@ gs_plugin_destroy (GsPlugin *plugin)
 
 	g_free (priv->personality);
 	g_free (priv->product_name);
+	g_free (priv->inst);
```

Comment:
`FlatpakInstallation` is a GObject, so you must use `g_object_unref()` with it, rather than `g_free()`.

Reply:
Although it can be `NULL` if the `get_user_installation()` call failed, so you should use `g_clear_object (&priv->inst)`.